qir/qsharp-target-tests/TargetTestScripts/__init__.py

```diff
@@ -1,63 +1,509 @@
+    "Decompositions/MResetZ.qs",
+    "Decompositions/R.qs",
+    "Decompositions/RAdjoint.qs",
+    "Decompositions/RControlled.qs",
+    "Decompositions/RAdjointControlled.qs",
+    "Decompositions/RControlledAdjoint.qs",
+    "Decompositions/R1.qs",
+    "Decompositions/R1Adjoint.qs",
+    "Decompositions/R1Controlled.qs",
+    "Decompositions/R1AdjointControlled.qs",
+    "Decompositions/R1ControlledAdjoint.qs",
+    "Decompositions/R1Frac.qs",
+    "Decompositions/R1FracAdjoint.qs",
+    "Decompositions/R1FracControlled.qs",
+    "Decompositions/R1FracAdjointControlled.qs",
+    "Decompositions/R1FracControlledAdjoint.qs",
+    "Decompositions/RFrac.qs",
+    "Decompositions/RFracAdjoint.qs",
+    "Decompositions/RFracControlled.qs",
+    "Decompositions/RFracAdjointControlled.qs",
+    "Decompositions/RFracControlledAdjoint.qs",
+    "Decompositions/Reset.qs",
+    "Decompositions/ResetAdjoint.qs",
+    "Decompositions/ResetControlled.qs",
+    "Decompositions/ResetAdjointControlled.qs",
+    "Decompositions/ResetControlledAdjoint.qs",
+    "Decompositions/Rx.qs",
+    "Decompositions/RxAdjoint.qs",
+    "Decompositions/RxControlled.qs",
+    "Decompositions/RxAdjointControlled.qs",
+    "Decompositions/RxControlledAdjoint.qs",
+    "Decompositions/Ry.qs",
+    "Decompositions/RyAdjoint.qs",
+    "Decompositions/RyControlled.qs",
+    "Decompositions/RyAdjointControlled.qs",
+    "Decompositions/RyControlledAdjoint.qs",
+    "Decompositions/Rz.qs",
+    "Decompositions/RzAdjoint.qs",
+    "Decompositions/RzControlled.qs",
+    "Decompositions/RzAdjointControlled.qs",
+    "Decompositions/RzControlledAdjoint.qs",
+    "Decompositions/S.qs",
+    "Decompositions/SAdjoint.qs",
+    "Decompositions/SControlled.qs",
+    "Decompositions/SAdjointControlled.qs",
+    "Decompositions/SControlledAdjoint.qs",
+    "Decompositions/SWAP.qs",
+    "Decompositions/SWAPAdjoint.qs",
+    "Decompositions/SWAPControlled.qs",
+    "Decompositions/SWAPAdjointControlled.qs",
+    "Decompositions/SWAPControlledAdjoint.qs",
+    "Decompositions/SetToBasisState.qs",
+    "Decompositions/T.qs",
+    "Decompositions/TAdjoint.qs",
+    "Decompositions/TControlled.qs",
+    "Decompositions/TAdjointControlled.qs",
+    "Decompositions/TControlledAdjoint.qs",
+    "Decompositions/X.qs",
+    "Decompositions/XAdjoint.qs",
+    "Decompositions/XControlled.qs",
+    "Decompositions/XAdjointControlled.qs",
+    "Decompositions/XControlledAdjoint.qs",
+    "Decompositions/Y.qs",
+    "Decompositions/YAdjoint.qs",
+    "Decompositions/YControlled.qs",
+    "Decompositions/YAdjointControlled.qs",
+    "Decompositions/YControlledAdjoint.qs",
+    "Decompositions/Z.qs",
+    "Decompositions/ZAdjoint.qs",
+    "Decompositions/ZControlled.qs",
+    "Decompositions/ZAdjointControlled.qs",
+    "Decompositions/ZControlledAdjoint.qs",
+
+
+    "CodeBlocks/QubitArrays1.qs",
+    "CodeBlocks/QubitArrays2.qs",
+    # TODO(tfr): Does not compile "CodeBlocks/Range1.qs",
```

Comment:
What issue is this one?

Reply:
The issue is slicing using `...` - it is not really a qat problem, so I remove this test.